src/Toolkit/Toolkit/UI/Controls/UtilityNetworkTraceTool/UtilityNetworkTraceToolController.cs

```diff
@@ -326,7 +326,7 @@ private void OnGeoModelLoaded(object? sender, EventArgs e)
                     IsLoadingNetwork = true;
                     UtilityNetworks.Clear();
 
-                    var utilityNetworks = (sender is Map map ? map.UtilityNetworks : null) ?? throw new ArgumentException("No UtilityNetworks found.");
+                    var utilityNetworks = (sender is Map map ? map.UtilityNetworks : null) ?? throw new ArgumentException(Properties.Resources.GetString("UtilityNetworkTraceToolNoUtilityNetworks"));
```

Comment:
Exception messages shouldn't be localized - only UI displayed to the end user. Goes for everything below here as well.
I'm also a little confused about throwing here. It looks to me like this is an `async void`, and async voids should never be allowed to throw

Reply:
Ah you're right exception here is not part of UI (only [Trace](https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/blob/main/src/Toolkit/Toolkit/UI/Controls/UtilityNetworkTraceTool/UtilityNetworkTraceToolController.cs#L689) errors are displayed to user) . It seems this one is wrapped in a try-catch that only marks readiness of UI. I can update this to null/empty check instead.

Reply:
I left the resource key because it's used in all XAML, say UN collection is cleared or empty.
![image](https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/assets/63615517/9a491bf3-969b-4d92-be49-16cfe3cd04ba)

Reply:
Actually, is it okay to put back the catch all? Should utility network fail to load when no credential for example, then the try-catch handle it and mark this control as not ready.

Reply:
I think fixing how we handle exceptions here is a bit out of scope for the specific issue, but something to look into - the main thing to change for now is to not add localizations for exceptions.